apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -280,10 +284,6 @@ function initializeBlocklyWrapper(blocklyInstance) {
     blockspace.addChangeListener(handler);
   };
 
-  blocklyWrapper.getWorkspaceCode = function() {
```

Comment:
This was removed because it was actually duplicated above in the same file: https://github.com/code-dot-org/code-dot-org/blob/dceaaceb0857f6f77cfc02ebcf877d3977cd16b6/apps/src/blockly/googleBlocklyWrapper.js#L112-L114

Reply:
nit: Maybe alphabetize functions in lines 99-113.

Reply:
There's a lot of cleanup we'd like to do in this file, so I'm going to hold off on this specific task for now.